package.json

```diff
@@ -84,6 +84,7 @@
   "dependencies": {
     "@aws-sdk/client-sqs": "^3.744.0",
     "@ministryofjustice/frontend": "^3.3.1",
+    "@ministryofjustice/hmpps-digital-prison-reporting-frontend": "^3.37.7",
```

Comment:
you don’t need to directly include transitive dependencies since you don’t actually use them directly so the only line you needed to dependencies was this one